src/scrapers/game_details_scrape.py

```diff
@@ -32,12 +32,26 @@ def soccer_summary(box_score_section):
     if scoring_section:
         scoring_rows = scoring_section.find(TAG_TBODY)
         if scoring_rows:
+            cornell_score = 0
+            opp_score = 0
             for row in scoring_rows.find_all(TAG_TR):
                 time = row.find_all(TAG_TD)[0].text.strip()
                 team = row.find_all(TAG_TD)[1].find(TAG_IMG)[ATTR_ALT]
                 event = row.find_all(TAG_TD)[2]
                 desc = event.find_all(TAG_SPAN)[-1].text.strip()
-                summary.append({'time': time, 'team': team, 'description': desc})
+                
+                if team == "COR" or team == "CU":
+                    cornell_score += 1
+                else:
+                    opp_score += 1
+                    
+                summary.append({
+                    'time': time, 
```

Comment:
should the time fields in all the different summary functions also be converted to UTC?